src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsKruxObject.java

```diff
@@ -54,14 +54,17 @@ public void verifyKruxControlTag(String kruxSiteId) {
    */
   public void verifyKruxUserParam(String slotName) {
     String script = "return localStorage.kxuser;";
-    waitForKrux();
     String user1 = (String) ((JavascriptExecutor) driver).executeScript(script);
+
+    // Fourth page view
     refreshPage();
-    waitForKrux();
+
     String user2 = (String) ((JavascriptExecutor) driver).executeScript(script);
     String gptPageParams = getGptPageParams(slotName);
+
     PageObjectLogging.log("gpt page params", gptPageParams, true);
     PageObjectLogging.log("krux users", user1 + ", " + user2, true);
+
     // TODO: figure out why we get krux user id in GPT calls from localStorage.kxuser in current PV OR from previous PV
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)